### ISSUE TITLE
support mustache?

### ISSUE THREAD

**Issue by aslak01** [g-plane/markup_fmt#65]
There are dozens of us out here still using mustache templates. I've tried several different alternatives that all nearly support formatting it consistently for me, but nothing quite works. It seems like something this project would potentially be able to support?

**Comment by g-plane** [comment]
Sorry, I don't have time to support new syntax, but let's keep this issue open.

**Comment by aslak01** [comment]
Of course, totally understandable. I see djlint is receiving updates again, so maybe I'll give that one another try, but last time it wasn't working very well for us, and some devs complained about the complexity of managing python environments. So if anyone who comes across this thread has any recommendations, let me know.
